src/main.ts

```diff
@@ -105,3 +108,17 @@ function resolveVersionInput(): string {
 
   return version;
 }
+
+async function printEnvDetails() {
+  // Output version of node and npm that are being used
+  await exec.exec('node', ['--version']);
+
+  // Older versions of Node don't include npm, so don't let this call fail
+  await exec.exec('npm', ['--version'], {
+    ignoreReturnCode: true
+  });
+
+  await exec.exec('yarn', ['--version'], {
```

Comment:
probably yarn is not need as it doesn't go with node, is that correct?

Reply:
Both GitHub's runners and corepack (if enabled) will automatically install Yarn